src/strategy/dungeons/wotlk/forgeofsouls/FosTriggers.h

```diff
@@ -0,0 +1,40 @@
+#ifndef _PLAYERBOT_WOTLKDUNGEONFOSTRIGGERS_H
+#define _PLAYERBOT_WOTLKDUNGEONFOSTRIGGERS_H
+
+#include "Trigger.h"
+#include "PlayerbotAIConfig.h"
+#include "GenericTriggers.h"
+#include "DungeonStrategyUtils.h"
+
+enum FosIDs
```

Comment:
Should be expanded to avoid conflicts in the future as this is not in it's own namespace